app/base/routes.py

```diff
@@ -79,12 +81,22 @@ def register():
 @blueprint.route("/account", methods=["GET", "POST"])
 @login_required
 def account():
+    from app.tasks import image_process, delete_img_obj
+
     user_id = current_user.id
     form = UpdateAccountForm()
     if form.validate_on_submit():
         if form.picture.data:
-            image = form.picture.data
-            save_profile_picture(image)
+            # delete previous image
+            delete_img_obj.delay(s3_bucket, s3_user_img_dir, filename=current_user.photo)
+
+            image = request.files["picture"]
+            redis_img_key = save_to_redis([image])
+            rand_str = str(uuid.uuid4())[:13].replace("-", "") + ".jpg"
```

Comment:
perhaps something like this here
`rand_str = f"{uuid.uuid4().__str__()[:6])}.jpg}"`

Reply:
I have updated the code to get the file extension using `os.path.splitext()` add concatenate it together with the generated file name.